README.md

```diff
@@ -280,10 +280,10 @@ You can learn more by visiting [the Blockstack Website](https://blockstack.org)
 
 You can also read the technical papers:
 
-- ["PoX: Proof of Transfer Mining with Bitcoin"](https://blockstack.org/pox.pdf), May 2020
-- ["The Blockstack Decentralized Computing Network"](https://blockstack.org/whitepaper.pdf), May 2019
+- ["PoX: Proof of Transfer Mining with Bitcoin"](https://assets.website-files.com/5fcf9ac604d37418aa70a5ab/60072dbb32d416d6b3806935_5f1596b12bcc0800f3dcadcd_pox.pdf), May 2020
```

Comment:
Are there better links for the PoX whitepaper? Is this hosted anywhere on `stacks.org` @jcnelson?

Reply:
This:  https://community.stacks.org/pox. ?

Reply:
They are different versions of the document.
The link to the pdf file in the site https://community.stacks.org/pox displays the older draft version of February 2020 (it shows an image of May-2020 but if you click on Download you will get the February draft). The link I provided is to the final document of May 2020. I have only found the file in the link provided.

Ideally, the link of https://blockstack.org/pox.pdf should aim to the final version of May and not the draft of February.

Reply:
Let's go ahead and use https://community.stacks.org/pox and update the paper hosted there.

Reply:
I agree that's the best option. 
I changed the link to the one mentioned and pushed the change.

Reply:
The paper at that link has been updated